packages/webview_flutter/webview_flutter_wkwebview/example/ios/RunnerUITests/FLTWebViewUITests.m

```diff
@@ -18,6 +42,97 @@ - (void)setUp {
   [self.app launch];
 }
 
+- (void)testTransparentBackground {
+  XCUIApplication* app = self.app;
+  XCUIElement* menu = app.buttons[@"Show menu"];
+  if (![menu waitForExistenceWithTimeout:30.0]) {
+    os_log_error(OS_LOG_DEFAULT, "%@", app.debugDescription);
+    XCTFail(@"Failed due to not able to find menu");
+  }
+  [menu tap];
+
+  XCUIElement* transparentBackground = app.buttons[@"Transparent background example"];
+  if (![transparentBackground waitForExistenceWithTimeout:30.0]) {
+    os_log_error(OS_LOG_DEFAULT, "%@", app.debugDescription);
+    XCTFail(@"Failed due to not able to find Transparent background example");
+  }
+  [transparentBackground tap];
+
+  XCUIElement* transparentBackgroundLoaded =
+      app.webViews.staticTexts[@"Transparent background test"];
+  if (![transparentBackgroundLoaded waitForExistenceWithTimeout:30.0]) {
+    os_log_error(OS_LOG_DEFAULT, "%@", app.debugDescription);
+    XCTFail(@"Failed due to not able to find Transparent background test");
+  }
+
+  XCUIScreenshot* screenshot = [[XCUIScreen mainScreen] screenshot];
+
+  UIImage* screenshotImage = screenshot.image;
+  UIColor* centerLeftColor = getPixelColorInImage(
+      screenshotImage, 0, (screenshotImage.scale * screenshotImage.size.height) / 2);
+  UIColor* centerColor = getPixelColorInImage(
+      screenshotImage, (screenshotImage.scale * screenshotImage.size.width) / 2,
+      (screenshotImage.scale * screenshotImage.size.height) / 2);
+  CIColor* centerLeftCIColor = [CIColor colorWithCGColor:centerLeftColor.CGColor];
+  CIColor* centerCIColor = [CIColor colorWithCGColor:centerColor.CGColor];
```

Comment:
Why are you converting from CGColors to CIColors to compare rather than using `CGColorEqualToColor`?